views/physicalobject.py

```diff
@@ -24,9 +24,9 @@ def zoom(coordinates: Sequence[Vector2], scale_factor: float, zoom_level: float)
 
 
 def project(coordinates: Sequence[Vector3], normalVector: Vector3) -> list[Vector2]:
-    """Project the 3D coordinates onto a 2D plane."""
-
-    a, b, c = 2*math.sin(normalVector.x), 2*math.sin(normalVector.y), normalVector.z
+    """Project the 3D coordinates onto a 2D plane."""  
+    normalVector = normalVector.normalize()
```

Comment:
I would expect a parameter that's called normalVector to already be normalized, see above.